### DISCUSSION TITLE
Other language support (non-english)

### DISCUSSION THREAD

**Discussion by pixelfenix** [Goldenfreddy0703/Otaku#17] [Ideas]
Hi! The plugin is amazing and super fast good job! Similarly to Kaito Beta (https://github.com/Goldenfreddy0703/plugin.video.kaito.beta/issues/36) I would like to humbly request that an option be added for Nyaa to parse non-english results. Nyaa is actually an amazing source for all types of regional subs however the URL used to parse currently specifically specifies english-only.

If I were better at coding I'd just do a push request of what jkaw did in Kaito but I'm afraid I'll just mess it up

**Comment by Goldenfreddy0703** [comment]
Hey there, I think it's a great idea cause I have not messed with this option in Kaito beta and honestly, I didn't even know it was an option in the first place. 😅

If possible we will add this to our todo list or we could probably achieve this today if we have time. Thanks for the idea and I'm surprised there was an option for that.

**Reply by pixelfenix** [reply]
(ﾉ◕ヮ◕)ﾉ*:･ﾟ✧

**Comment by Goldenfreddy0703** [comment]
Hey so do you still want this, i forgot. 😅

**Reply by pixelfenix** [reply]
I would say it's less needed in today's world where the biggest sources are often include subs in a multitude of languages. It would probably be needed when searching for older content though.

**Reply by Goldenfreddy0703** [reply]
true, alright will close this cause we have Hianime for that and all.
